registry/auth/token/token.go

```diff
@@ -144,8 +147,14 @@ func (t *Token) Verify(verifyOpts VerifyOptions) error {
 
 	// Verify that the token is currently usable and not expired.
 	currentUnixTime := time.Now().Unix()
-	if !(t.Claims.NotBefore <= currentUnixTime && currentUnixTime <= t.Claims.Expiration) {
-		log.Errorf("token not to be used before %d or after %d - currently %d", t.Claims.NotBefore, t.Claims.Expiration, currentUnixTime)
+
+	if t.Claims.Expiration <= currentUnixTime-int64(Leeway.Seconds()) {
```

Comment:
I find this easier to read as `if currentUnixTime >= t.Claims.Expiration + int64(Leeway.Seconds()`, but that might just be me.